src/marks/tip.js

```diff
@@ -0,0 +1,237 @@
+import {select} from "d3";
+import {create} from "../context.js";
+import {formatDefault} from "../format.js";
+import {Mark} from "../mark.js";
+import {maybeFrameAnchor, maybeKeyword, maybeTuple, number, string} from "../options.js";
+import {applyChannelStyles, applyDirectStyles, applyIndirectStyles} from "../style.js";
+import {applyFrameAnchor, applyTransform} from "../style.js";
+import {inferTickFormat} from "./axis.js";
+import {applyIndirectTextStyles, cut, defaultWidth, monospaceWidth} from "./text.js";
+
+const defaults = {
+  ariaLabel: "tip",
+  fill: "white",
+  stroke: "currentColor"
+};
+
+export class Tip extends Mark {
+  constructor(data, options = {}) {
+    const {
+      x,
+      y,
+      x1,
+      x2,
+      y1,
+      y2,
+      anchor,
+      monospace,
+      fontFamily = monospace ? "ui-monospace, monospace" : undefined,
+      fontSize,
+      fontStyle,
+      fontVariant,
+      fontWeight,
+      lineHeight = 1,
```

Comment:
lineHeight separates fields, and this default makes the tooltip feel a bit cramped; lineHeight: 1.3 would give more breathing room, or we could separate lineHeight (for text contents) and padding (to separate fields)?

Reply:
I want to use the same default as the text mark.